src/material-experimental/mdc-table/cell.ts

```diff
@@ -52,16 +52,13 @@ export class MatFooterCellDef extends CdkFooterCellDef {}
  */
 @Directive({
   selector: '[matColumnDef]',
-  inputs: ['sticky'],
+  inputs: ['sticky', 'name: matColumnDef'],
```

Comment:
Will this work with Closure Compiler's property renaming?
(same for similar changes throughout)

Reply:
I think for now I’ll prefer to go with my initial approach to override the accessor, it does look like this also breaks VE unless I am doing something wrong.

Reply:
I'm not sure how Closure would be affected by this. I wonder why View Engine breaks. We use that pattern in similar cases I think:

https://cs.opensource.google/angular/components/+/master:src/cdk/table/row.ts;l=95?q=inputs:&ss=angular%2Fcomponents&start=11

Reply:
Maybe it doesn’t work with accessors?

This change https://github.com/angular/components/pull/20372 causes failures in VE